### PR TITLE
 Fix merge_label utility argument order 

### PR DESCRIPTION
**Pull request by Ladsgroup**
Bug: https://phabricator.wikimedia.org/T187742

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -34,7 +34,6 @@ models: \
 		lvwiki_models \
 		nlwiki_models \
 		nowiki_models \
-		nowiki_models \
```
Comment (Ladsgroup) [inline Makefile:37]: I don't know how it got there but it's redundant and the auto-generation removed it, same for other removals.

File: Makefile [hunk 2]
```diff
@@ -550,7 +546,7 @@ datasets/enwiki.human_labeled_revisions.20k_2015.json:
 		https://labels.wmflabs.org/campaigns/enwiki/4/ > $@
 
 datasets/enwiki.labeled_revisions.20k_2015.json: \
-		datasets/enwiki.human_labeled_revisions.20k_2015.json
+		datasets/enwiki.human_labeled_revisions20k_2015.json
```
Comment (halfak) [inline Makefile]: Missing a "."

File: templates/Makefile.j2 [hunk 3]
```diff
@@ -74,9 +74,9 @@ datasets/{{ wiki.name }}.revisions_for_review.{{ wiki.review_sample }}.json: \
 	grep '"needs_review": true' $< | shuf > $@
 {% endif %}{% if wiki.merged_samples %}{% for big_sample_name, big_sample in wiki.merged_samples|dictsort %}
 datasets/{{ wiki.name }}.labeled_revisions.{{ big_sample_name }}.json: \
-{%- for sample_name, sample in big_sample|dictsort %}
-		datasets/{{ wiki.name }}.{{ sample }}.{{ sample_name }}.json{% if not loop.last %} \{% endif -%}
-{% endfor %}
+{%- if big_sample.human_labeled_revisions %}
+		datasets/{{ wiki.name }}.human_labeled_revisions{{ big_sample.human_labeled_revisions }}.json{%- if big_sample.autolabeled_revisions %} \
```
Comment (halfak) [inline templates/Makefile.j2]: I think you need to add the "." here

**Comment by Ladsgroup** [comment]
Done, so stupid (facepalm)

**Pull request merged**